mantidimaging/gui/windows/spectrum_viewer/model.py

```diff
@@ -191,6 +192,40 @@ def save_csv(self, path: Path, normalized: bool) -> None:
 
         with path.open("w") as outfile:
             csv_output.write(outfile)
+        self.save_roi_coords(self.get_roi_coords_filename(path))
+
+    def get_roi_coords_filename(self, path: Path) -> Path:
+        """
+        Get the path to save the ROI coordinates to.
+
+        @param path: The path to save the CSV file to.
+        @return: The path to save the ROI coordinates to.
+        """
+        return path.with_stem(f"{path.stem}_roi_coords")
+
+    def save_roi_coords(self, path: Path) -> None:
+        """
+        Save the coordinates of the ROIs to a csv file (ROI name, x_min, x_max, y_min, y_max)
+        following Pascal VOC format.
+
+        @param path: The path to save the CSV file to.
+        """
+        roi_coords: dict[str, list[int]] = {}
+        roi_coords.update(
+            {roi_name: [roi.left, roi.top, roi.right, roi.bottom]
+             for roi_name, roi in self._roi_ranges.items()})
+
+        with open(path, encoding='utf-8', mode='w') as f:
+            csv_writer = csv.DictWriter(f, fieldnames=["ROI", "X Min", "X Max", "Y Min", "Y Max"])
+            csv_writer.writeheader()
+            for roi_name, coords in roi_coords.items():
```

Comment:
Can we just iterate over `self._roi_ranges.items()` here?

Reply:
Yes, you're right! nice catch! For some reason, iterating through `_roi_ranges` just didn't cross my mind 😅